src/styles/main.css

```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
I'm thinking tailwind thik chha but should be an optional dependency and we should provide a way to let users choose their own library/solution for styling

Reply:
this could be a follow up worker

Reply:
this only for package itself we'dont force user to use tailwind user can use whatever css  they want